spec/fixtures/multimedia/file_manifest.csv

```diff
@@ -13,7 +13,7 @@ object2,bb222bb2222_002_a_sh.wav,,,no,yes,no,,
 object2,bb222bb2222_002_a_sl.mp3,,,yes,yes,yes,,
 object2,bb222bb2222_002_img_1.jpg,,,yes,yes,yes,,
 object2,bb222bb2222_002_b_pm.wav,"Tape 1, Side B",2,no,yes,no,media,
-object2,bb222bb2222_002_b_sh.wav,,,no,no,no,,
+object2,bb222bb2222_002_b_sh.wav,,,no,yes,no,,
 object2,bb222bb2222_002_b_sl.mp3,,,yes,yes,yes,,
```

Comment:
these existing fixtures are not actually valid anymore and need to be changed (and the ones below as well)... we cannot have "no, no, no" anymore for publish,preserve, shelve